package.json

```diff
@@ -6,10 +6,10 @@
   "private": true,
   "scripts": {
     "prepare": "husky install && turbo run prepare && bash scripts/print_dundring.sh",
-    "start": "FORCE_COLOR=1 turbo run start --parallel",
-    "mock": "FORCE_COLOR=1 turbo run mock --parallel",
```

Comment:
Tested the start command as well – failing on first try without having built the types. This is true both with and without the `--parallel` flag, so i chose to remove it. 

Solved the problem by including `yarn build` to the `prepare` script of `@dundring/types` – shabby, but not too bad imo